packages/client/hmi-client/src/components/widgets/tera-modal.vue

```diff
@@ -54,6 +62,14 @@ onMounted(() => {
 			detail: { id: element.id ?? '' }
 		});
 		element.dispatchEvent(event);
+
+		// Focus the first input element in the default slot
+		nextTick(() => {
+			const input = element.querySelector('section.content input') ?? element.querySelector('section.content textarea');
+			if (input) {
+				(input as HTMLElement).focus();
+			}
```

Comment:
Why not use a ref here?

Reply:
A ref is actually used but I guess it's not as obvious due to the variable name, I shall fix